Sources/SwiftProtobuf/Visitor.swift

```diff
@@ -690,4 +692,8 @@ extension Visitor {
   public mutating func visitExtensionFields(fields: ExtensionFieldValueSet, start: Int, end: Int) throws {
     try fields.traverse(visitor: &self, start: start, end: end)
   }
+    
+    func shouldIncludeDefault() -> Bool {
```

Comment:
Indent is off compared to the rest of the file, and this likely should get a comment since unlike the rest of these, it isn't remapping to another call, it is providing a default implementation.